docs/manage/connections-integrations/webhook-connections/set-up-webhook-connections.md

```diff
@@ -50,10 +52,11 @@ To set up a webhook connection:
     :::
 1. (Optional) If the third-party system requires an **Authorization Header**, enter it here. For more information, see [Example Authorization Header](#example-authorization-header) below.
 1. (Optional) **Custom Headers**, enter up to five comma separated key-value pairs.
-1. For **Payload**, enter a JSON object in the format required by the target webhook URL. For details on variables that can be used as parameters within your JSON object, see [webhook payload variables](#webhook-payload-variables), below. When using a generic webhook connection for notifications in a [Monitor](/docs/alerts/monitors) you can distinguish between an alert and a recovery by using the [alert variable](../../../alerts/alert-variables.md) `TriggerType`. An alert will have a value of either `Critical` , `Warning`, or `Missing ``Data`. Recovery will have either `CriticalResolved`, `WarningResolved`, or `Missing Data Resolved`. The same payload for both alerts and recovery is used.
+1. Under **Alert Payload**, enter a JSON object in the format required by the target webhook URL. For details on variables that can be used as parameters within your JSON object, see [webhook payload variables](#webhook-payload-variables), below. When using a generic webhook connection for notifications in a [Monitor](/docs/alerts/monitors) you can distinguish between an alert and a recovery by using the [alert variable](../../../alerts/alert-variables.md) `TriggerType`. An alert will have a value of `Critical`, `Warning`, or `Missing Data`. Recovery will have `CriticalResolved`, `WarningResolved`, or `Missing Data Resolved`. <br/><img src={useBaseUrl('img/connection-and-integration/alert-payload.png')} alt="recovery payload" />
     :::note
     Variables are escaped according to the JSON standard, meaning that they can be used in application JSON. 
     :::
+1. (Optional) Under **Recovery Payload**, you can customize your resolution notification payload JSON.<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload.png')} alt="recovery payload" /><br/>Example:<br/><img src={useBaseUrl('img/connection-and-integration/recovery-payload-example.png')} alt="recovery payload" />
 1. Click **Save**.
```

Comment:
There are also two button at the bottom of the page.

Test Alert Payload 
Test Alert + Recovery Payload 

We should mention what these buttons do. 

As the name suggests, these buttons allow user to test their alert and recovery payload after they have customized it. When a user clicks these buttons, Sumo Logic fires a sample webhook call to the given endpoint with the mentioned payload.

Reply:
Thank you! Will note this.